guide/locations/_azure-ARM.md

```diff
@@ -158,9 +158,8 @@ brooklyn.catalog:
       destroyOnFailure: false
 
       templateOptions:
-      overrideLoginUser: brooklyn
-      overrideLoginPassword: "secretPass1!"
-      resourceGroup: <ResourceGroup-name>
```

Comment:
Yes, for Linux Azure ARM locations Apache Brooklyn docs does not advice to use a specific `resourceGroup`. I do not see a reason why we should advice users to use fixed resourceGroup.
I tested without this parameter and it works.